imageroot/systemd/user/roundcubemail-app.service

```
@@ -16,6 +16,7 @@ Restart=always
 TimeoutStopSec=70
 ExecStartPre=/bin/mkdir -p %S/state/config
 ExecStartPre=/bin/rm -f %t/roundcubemail-app.pid %t/roundcubemail-app.ctr-id
+ExecStartPre=/usr/local/bin/runagent discover-service
```

What to do if the discover-service script fails?

1. abort the unit startup
2. add `-` prefix, ignore its exit code and start anyway with the old configuration, provided it was not overwritten

```suggestion
ExecStartPre=-/usr/local/bin/runagent discover-service
```

---

done